src/main/kotlin/com/devexperts/openhack2022/cell_sandbox/gui/GenomeEditingPanel.kt

```diff
@@ -0,0 +1,9 @@
+package com.devexperts.openhack2022.cell_sandbox.gui
+
+import javafx.scene.layout.Pane
+
+class GenomeEditingPanel: Pane() {
+    init {
```

Comment:
Do we need an empty constructor?

Reply:
Oh, forgot about that. Just a temporary thing